tests/apps/conftest.py

```diff
@@ -13,28 +13,28 @@
 @pytest.fixture
 def balsamic_orderform(orderforms: Path) -> str:
     """Orderform fixture for Balsamic samples"""
-    _file = orderforms / "1508.26.balsamic.xlsx"
+    _file = orderforms / "1508.27.balsamic.xlsx"
```

Comment:
Why not make 27 (the version I presume) a test constant so that you only need to update in one place and it is more readable

Reply:
Fixed with new constant